src/ESM.sol

```diff
@@ -65,6 +72,35 @@ contract ESM {
         require(z >= x);
     }
 
+    // --- Auth ---
+    mapping (address => uint256) public wards;
```

Comment:
I'd say we move this mapping to the slot 0 that has been quite a standard excepting for a few cases.

Reply:
oh yeah, good call